sqlalchemy_utils/i18n.py

```diff
@@ -12,13 +12,15 @@
 except ImportError:
     babel = None
 
+
 try:
-    from flask_babel import get_locale
-except ImportError:
+    def get_locale(): return babel.Locale('en')
+except AttributeError:
+    # As babel is optional, we may raise an AttributeError accessing it
     def get_locale():
         raise ImproperlyConfigured(
             'Could not load get_locale function from Flask-Babel. Either '
```

Comment:
Now that `Flask-Babel` is no longer required I guess you can remove mention of it in the error message.